Formula/nethack.rb

```diff
@@ -33,6 +35,13 @@ def install
                 /^HACKDIR=.*/,
                 "HACKDIR=#{libexec}"
 
+      if MacOS.version >= :mojave
+        # Also build the new curses interface
+        inreplace "hints/#{hintfile}",
+                  /^#WANT_WIN_CURSES=1$/,
+                  "WANT_WIN_CURSES=1"
```

Comment:
Is it possible to just set this environment variable below? `inreplace` can be fragile.

Reply:
`inreplace` certainly can be fragile. That's why I made sure to replace only a whole line match. Sadly, IIUC, this isn't really an environment variable but instead a Makefile variable and is designed to be edited in place (like `HACKDIR` above). Please correct me if I'm wrong. :)

Reply:
`make` populates all environment variables as Make variables when it is run.